opentech/apply/utils/image.py

```diff
@@ -1,12 +1,22 @@
+from django.core.cache import cache
 from django.urls import reverse
 from django.utils.html import format_html
 
 
+def image_url_cache_key(image_id, spec):
+    return f'image_url_cache_{image_id}_{spec}'
+
+
 def generate_image_url(image, filter_spec):
+    cache_key = image_url_cache_key(image.id, filter_spec)
```

Comment:
Since this is a single line function pull the string it builds to this line (and ditch the function).  You'll improve readability by putting the magic string near where it's used.

Reply:
I was expecting to use it twice then didn't. You are correct.